docs/misc/nsquitalwayskeepwindows.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Keep windows when quitting an application | Miscellaneous
+description: 
+head:
+  - - meta
+    - property: 'og:title'
+      content: macOS defaults > Miscellaneous > Keep windows when quitting an application
+  - - meta
+    - property: 'og:description'
+      content: When enabled, open documents and windows will be restored when you re-open an application.
+---
+
+# Keep windows when quitting an application
+
+When enabled, open documents and windows will be restored when you re-open an application.
+This setting is the inverse of the option accessible through
+System Preferences > Desktop & Dock > Close windows when quitting an application.
```

Comment:
Please move this part to a "Set value from UI" section at the end of the page. Take a look at [an example](https://github.com/yannbertrand/macos-defaults/blob/b4cc05a76c9459d389b8916e240a1337962045bc/docs/keyboard/applekeyboarduimode.md?plain=1#L60).

Reply:
Done, see 6afc85c488a0cfaab6b1820ee9674a820f7fae8e